api/src/main/java/org/arquillian/cube/q/api/NetworkChaos.java

```diff
@@ -0,0 +1,138 @@
+package org.arquillian.cube.q.api;
+
+import java.util.concurrent.TimeUnit;
+
+public interface NetworkChaos {
+
+    Action on(String machine, int port);
+
+    public interface Action extends Q {
```

Comment:
Generally speaking would be great to spend some time on Javadoc, as it's public-facing API we have here.